iothub/service/src/Common/Data/SharedAccessSignatureAuthorizationRule.cs

```diff
@@ -37,9 +47,17 @@ public string SecondaryKey
             }
         }
 
+        /// <summary>
+        /// The name of the shared access policy that will be used to grant permission to IoT Hub endpoints.
+        /// </summary>
         [JsonProperty(PropertyName = "rights")]
         public AccessRights Rights { get; set; }
 
+        /// <summary>
+        /// Evaluates if two <see cref="SharedAccessSignatureAuthorizationRule"/> objects have the same shared access policies and keys.
+        /// </summary>
+        /// <param name="other">The <see cref="SharedAccessSignatureAuthorizationRule"/> object to compare the current <see cref="SharedAccessSignatureAuthorizationRule"/> to.</param>
```

Comment:
Seems redundant

Reply:
how would you suggest I phrase it?

Reply:
You don't need to add links to the type twice, for one. I'd be fine with that.

Reply:
So I ended up removing the doc comments, since it seemed self-explanatory and I had also modified the type to be internal.